tests/test_rotation.c

```diff
@@ -0,0 +1,535 @@
+/**
+ * @file test_aging.c
+ * @author Ondrej Kusnirik <Ondrej.Kusnirik@cesnet.cz>
+ * @brief tests for rotation utility
+ *
+ * @copyright
+ * Copyright (c) 2018 - 2022 Deutsche Telekom AG.
+ * Copyright (c) 2018 - 2022 CESNET, z.s.p.o.
+ *
+ * This source code is licensed under BSD 3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#define _GNU_SOURCE
+
+#include <stdio.h>
+#include <string.h>
+#include <dirent.h>
+#include <unistd.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+#include <signal.h>
+#include <errno.h>
+#include <stdlib.h>
+#include <setjmp.h>
+#include <stdarg.h>
+#include <cmocka.h>
+
+#include <libyang/libyang.h>
+
+#include "sysrepo.h"
+#include "sysrepo_types.h"
+#include "test_common.h"
+#include "config.h"
+
+#define MAX_WAIT_MICROSEC 100000
```

Comment:
I did not like the name because `MAX_WAIT` says nothing about where this value is used. Something like `TIMEOUT_STEP_MS` would be better.

Reply:
Changed